src/utils/stringFormat.tsx

```diff
@@ -176,6 +177,29 @@ export function parseLatLng(str: string): Coordinate | null {
   return [lat, lng];
 }
 
+export function generateTreeCareMessage(item: Activity): string {
+  const activityStrings = [] as string[];
+  if (item.cleaned) activityStrings.push('cleared of waste');
+  if (item.mulched) activityStrings.push('mulched');
+  if (item.watered) activityStrings.push('watered');
+  if (item.weeded) activityStrings.push('weeded');
```

Comment:
Great job! I think that something to look into is marking stewardship activities as icons down the line. For now this formatting is great, but as we add functionality like delete/edit stewardship activities, having icons vs long activity messaging might look cleaner

Reply:
Yeah that's a really good idea! We might have to make a couple icons for the more tree-specific activities like mulching and weeding.